tests/components/deconz/test_climate.py

```diff
@@ -347,7 +347,7 @@ async def test_climate_device_with_cooling_support(
             "0": {
                 "config": {
                     "battery": 25,
-                    "coolsetpoint": None,
+                    "coolsetpoint": 1111,
```

Comment:
Change of behaviour? (See comment above)

Reply:
This is to address the changed test coverage for the target_temperature method

Reply:
ok